react/components/Metrics/DataAnalysis/CustomElements/CustomXAxisTick.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+
+interface Props {
+  x: number
+  y: number
+  payload: any
```

Comment:
👀

Reply:
In the @types/recharts I didn't find a interface that have those props